test/c/test_main.c

```diff
@@ -137,7 +137,7 @@ int test_num_qubits()
     return 0;
 }
 
-int main()
+int test_main()
```

Comment:
This is the only test that's run because it has the same name as the file, did I understand that correctly? So we'd need a "master test function" in each file that triggers all tests in a file?

Reply:
Yes correct